db/models/package.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Package = db.define('package', {
+  description: {
+    type: Sequelize.TEXT,
+    allowNull: false
+  },
+  image: {
```

Comment:
As a convention, we're calling the image field `imageURL` and adding a validation to check that it is a URL. 
```javascript 
  imageURL: {
    type: Sequelize.STRING,
    validate: {
      isUrl: true
    }
  }
```